scripts/lighthouseRunner.js

```diff
@@ -0,0 +1,88 @@
+const lighthouse = require('lighthouse');
+const chromeLauncher = require('chrome-launcher');
+const chalk = require('chalk');
+
+const log = console.log;
+const config = require(`./../lighthouse`);
+
+function launchChromeAndRunLighthouse(url, opts, config = null) {
+  return chromeLauncher
+    .launch({ chromeFlags: opts.chromeFlags })
+    .then(chrome => {
+      opts.port = chrome.port;
+      return lighthouse(url, opts, config).then(results =>
+        chrome.kill().then(() => results.lhr),
+      );
+    });
+}
+
+function getScoresPerCategory(reportCategories, audits, url) {
```

Comment:
`audits` and `url` don't need to be passed to this function, they're just returned in the object. This func could just return the mapped `scores`, and  injected into an object [here](https://github.com/BBC-News/simorgh/pull/876/files#diff-897fc147e049801433a6a6a5c986776fR46).